libraries/AdaptiveExpressions/TriggerTrees/Trigger.cs

```diff
@@ -118,6 +128,15 @@ public RelationshipType Relationship(Trigger other, Dictionary<string, IPredicat
             return result;
         }
 
+        /// <summary>
+        /// Determines whether is there any member in current clauses matches another Clause instance. 
```

Comment:
>Determines whether is there any member in current clauses matches another Clause instance. [](start = 12, length = 90)

Consider "Determines whether there is a member in the current Clause that matches the nodeClause parameter.